gpflow/utilities/utilities.py

```diff
@@ -228,12 +228,21 @@ def reset_cache_bijectors(input_module: tf.Module) -> tf.Module:
     target_types = (tfp.bijectors.Bijector, )
     accumulator = ('', None)
 
+    def clear_cache(b: tfp.bijectors.Bijector):
+        if isinstance(b, tfp.bijectors.Bijector):
```

Comment:
This is a bit confusing - the signature's type annotation does not make sense together with the isinstance check: if the isinstance is needed, then the types this function takes must be more than just tfp.bijectors.Bijector ... could you clarify?

Reply:
It can be any type